second_sem/hw5/5_4/EventLoop.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+
+namespace MyWorks
+{
+	public class EventLoop
+	{
+		private int positionY = Console.CursorLeft; // vertical cursor position
+		private int positionX = Console.CursorTop; // horisontal cursor position
+
+		// eventLoop body
+		public void Run()
+		{
+			while (true)
+			{
+				var key = Console.ReadKey();
+				switch (key.Key)
+				{
+				case ConsoleKey.LeftArrow:
+					{
+						if (positionX > 0)
+						{
+							positionX -= 1;
+							Console.SetCursorPosition(positionX, positionY);
```

Comment:
добавлю изменения, из нового только 11я и UML

2016-05-25 17:21 GMT+03:00 Yurii Litvinov <notifications@github.com>:

> In second_sem/hw5/5_4/EventLoop.cs
> <https://github.com/arthurnage/university-tasks/pull/6#discussion_r64581273>
> :
>
> > +		private int positionX = Console.CursorTop; // horisontal cursor position
> > +
> > +		// eventLoop body
> > +		public void Run()
> > +		{
> > +			while (true)
> > +			{
> > +				var key = Console.ReadKey();
> > +				switch (key.Key)
> > +				{
> > +				case ConsoleKey.LeftArrow:
> > +					{
> > +						if (positionX > 0)
> > +						{
> > +							positionX -= 1;
> > +							Console.SetCursorPosition(positionX, positionY);
>
> ну вот да, а то я хочу позакрывать всё, что зачтено, и посмотреть всё, что
> ещё не видел, а у Вас код везде старый
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/arthurnage/university-tasks/pull/6/files/27186ea0361bdd387fb566d75223781580810096#r64581273>
>